Source/Shaders/GlobeVS.glsl

```diff
@@ -38,13 +38,12 @@ float get2DMercatorYPositionFraction()
     float northLatitude = u_southAndNorthLatitude.y;
     if (northLatitude - southLatitude > maxTileWidth)
     {
-        float southMercatorYLow = u_southMercatorYLowAndHighAndOneOverHeight.x;
-        float southMercatorYHigh = u_southMercatorYLowAndHighAndOneOverHeight.y;
+        float southMercatorY = u_southMercatorYLowAndHighAndOneOverHeight.y;
```

Comment:
`u_southMercatorYLowAndHighAndOneOverHeight` doesn't need to store low and high now.  It can just store the double value casted to a float.

So it can be renamed to `u_southMercatorYAndOneOverHeight` and made a `vec2`.  Also, update the JavaScript code that sets this uniform.